### PR TITLE
Fix `NettyClient` invoke response

### PR DESCRIPTION
**Pull request by senyer**

### CODE REVIEW

File: transport/src/main/java/com/distkv/drpc/netty/NettyClient.java [hunk 1]
```diff
@@ -119,15 +118,15 @@ public Response invoke(Request request) {
       if (clientChannel.isActive()) {
         clientChannel.writeAndFlush(byteBuf).sync();
       } else {
-        throw new DrpcException("ClientChannel closed");
+        response.setThrowable(new DrpcException("ClientChannel closed"));
```
Comment (jovany-wang) [inline transport/src/main/java/com/distkv/drpc/netty/NettyClient.java]: If client channel is closed, why should we fill the response?

File: transport/src/main/java/com/distkv/drpc/netty/NettyClient.java [hunk 2]
```diff
@@ -119,15 +118,15 @@ public Response invoke(Request request) {
...
+        response.isError();
+        return response;
       }
       return response;
     } catch (Exception e) {
```
Comment (jovany-wang) [inline transport/src/main/java/com/distkv/drpc/netty/NettyClient.java]: It's broad to catch `Exception` here.

File: transport/src/main/java/com/distkv/drpc/netty/NettyClient.java [hunk 3]
```diff
@@ -119,15 +118,15 @@ public Response invoke(Request request) {
...
-      Response errorResponse = new ProtobufResponseDelegate();
-      errorResponse.setRequestId(request.getRequestId());
-      errorResponse
-          .setThrowable(new TransportException("NettyClient: response.getValue interrupted!"));
-      return errorResponse;
+      response.setThrowable(new TransportException("NettyClient: response.getValue interrupted!"));
+      response.isError();
```
Comment (zhengrenjie) [inline transport/src/main/java/com/distkv/drpc/netty/NettyClient.java]: this method no need be invoked here.

File: transport/src/main/java/com/distkv/drpc/netty/NettyClient.java (hunk 2 above)
Comment (senyer) [reply transport/src/main/java/com/distkv/drpc/netty/NettyClient.java]: yes，i agree

**Review by jovany-wang — APPROVED** [review]
LGTM

**Pull request merged**